### PR TITLE
fix: Update pipeline createRequest commit information and confignamespaces information when the project pipeline executing

### PR DESCRIPTION
**Pull request by kakj-go**
#### What this PR does / why we need it:
If the createrequest commit and confignamespace of the pipeline are not updated, the pre check of the execution pipeline will fail

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |        Update the commit and confignamespaces information when the project pipeline is executing     |
| 🇨🇳 中文    |       项目流水线执行的时候更新 commit 和 configNamespaces 信息       |

### CODE REVIEW

File: modules/dop/providers/projectpipeline/service_impl.go [hunk 1]
```diff
@@ -872,33 +911,92 @@ func (p *ProjectPipelineService) batchGetPipelineSources(pipelineSourceIDArray [
...
+
+	// update user gittar token
+	var worker = limit_sync_group.NewWorker(3)
+	worker.AddFunc(func(locker *limit_sync_group.Locker, i ...interface{}) error {
+		orgStr := createV2.Labels[apistructs.LabelOrgID]
+		orgID, err := strconv.ParseUint(orgStr, 10, 64)
+		if err != nil {
+			return apierrors.ErrRunProjectPipeline.InternalError(fmt.Errorf("not find orgID"))
+		}
+		// update CmsNsConfigs
```
Comment (sfwn) [inline modules/dop/providers/projectpipeline/service_impl.go:937]: update which config

File: modules/dop/providers/projectpipeline/service_impl.go [hunk 2]
```diff
@@ -872,33 +911,92 @@ func (p *ProjectPipelineService) batchGetPipelineSources(pipelineSourceIDArray [
...
+
+		locker.Lock()
+		defer locker.Unlock()
+
+		createV2.Labels[apistructs.LabelCommitDetail] = string(commitInfo)
+		return nil
+	})
+	// update createV2 ConfigManageNamespaces
+	worker.AddFunc(func(locker *limit_sync_group.Locker, i ...interface{}) error {
+		relationMap, err := p.AppSvc.GetPublishItemRelationsMap(apistructs.QueryAppPublishItemRelationRequest{AppID: int64(appDto.ID)})
```
Comment (sfwn) [inline modules/dop/providers/projectpipeline/service_impl.go:972]: strange, why need update publish-item's cms namespaces?

**Review by sfwn — CHANGES_REQUESTED** [review]

**Comment by sfwn** [comment]
reuse the logic of cicd please.

**Review by sfwn — CHANGES_REQUESTED** [review]
reuse the logic of cicd please.

File: modules/dop/providers/projectpipeline/service_impl.go (hunk 2 above)
Comment (kakj-go) [reply modules/dop/providers/projectpipeline/service_impl.go]: Changed back to the original ConvertPipelineToV2 method for reuse

File: modules/dop/providers/projectpipeline/service_impl.go (hunk 1 above)
Comment (kakj-go) [reply modules/dop/providers/projectpipeline/service_impl.go:902]: user clone code token config

File: modules/dop/providers/projectpipeline/service_impl.go [hunk 3]
```diff
@@ -910,6 +938,15 @@ func (p *ProjectPipelineService) autoRunPipeline(identityInfo apistructs.Identit
 	return value, nil
 }
 
+func getExtraValue(definition *dpb.PipelineDefinition) (*apistructs.PipelineDefinitionExtraValue, error) {
```
Comment (sfwn) [inline modules/dop/providers/projectpipeline/service_impl.go:941]: move this method to definition pkg

Comment (sfwn) [reply modules/dop/providers/projectpipeline/service_impl.go:941]: and you should use this new method in all places.

**Review by sfwn — CHANGES_REQUESTED** [review]

**Review by sfwn — APPROVED** [review]

**Pull request merged**